gradlew

```diff
@@ -44,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx2048m" "-Xms1024m"'
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
I'm not sure why you've changed this...

Reply:
I've never touched gradlew before so not sure why it changed either, but I have Reverted it back to what it should be